fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -77,6 +79,21 @@
             SYSTEM_CONFIG.getPackageVariableName("lucene_search_timeout_ms"), 600000
     );
 
+    public static final LRUQueryCache LUCENE_QUERY_CACHE;
+    static {
+        // Lucene 5.3.0 disables query caching.  Create cache for use here.
+        // min of 32MB or 5% of the heap size.  Same default as Lucene 5.4.0.
+        final long maxRamBytesUsed = SYSTEM_CONFIG.getLongProperty(
+            SYSTEM_CONFIG.getPackageVariableName("lucene_cache_size_bytes"),
+            Math.min(1L << 25, Runtime.getRuntime().maxMemory() / 20));
```

Comment:
It's better if closing parenthesis goes to the next line.

```
final long maxRamBytesUsed = SYSTEM_CONFIG.getLongProperty(
        SYSTEM_CONFIG.getPackageVariableName("lucene_cache_size_bytes"),
        Math.min(1L << 25, Runtime.getRuntime().maxMemory() / 20)
);
```

@cdeszaq Does fili refer 8 space indent on the next-line-continuation?